Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.9.15
```

Comment:
can we delete this from the root dir if we also save in the docker docker/base dir?

Reply:
Gotcha yea I think renaming for now makes sense